### PR TITLE
Citations and documentation improvements

### PR DESCRIPTION
**Pull request by erxleben**

### CODE REVIEW

**Comment by TorHou** [comment]
Wow very cool.

**Comment by bgruening** [comment]
@TorHou is this a +1? Review appreciated! :)

File: tools/glimmer3/glimmer_w_icm.xml [hunk 1]
```diff
@@ -192,7 +199,7 @@ The ICM can be generated with extracted CDS from related organisms (ICM builder)
 		orf00033     6217     6336  +1     3.09
 		........
 
-	- Glimmer detailed report (optional)
+output: Glimmer detailed report (optional)
```
Comment (TorHou) [inline tools/glimmer3/glimmer_w_icm.xml]: *Output 
comments by:
OCD Torsten

File: tools/glimmer3/glimmer_w_icm.xml [hunk 2]
```diff
@@ -135,18 +135,28 @@
...
-This is the main program that makes gene preditions based on an interpolated context model (ICM).
+This is the main program that makes gene predictions based on an interpolated context model (ICM).
 
 The ICM can be generated with extracted CDS from related organisms (ICM builder). If you can't generate an ICM model you can use the non knowlegde-based Glimmer with a de novo prediction.
 
------
+
+**Input**
+
+- interpolated context model (ICM): Use the 'Glimmer ICM builder' tool to create one
```
Comment (TorHou) [inline tools/glimmer3/glimmer_w_icm.xml]: Interpolate Context Model

File: tools/rna_tools/miRDeep2/miRDeep2/mirdeep2.xml [hunk 3]
```diff
@@ -184,9 +184,33 @@ automatic estimation of the optimal value and all detected precursors are analyz
...
+
+arf format:
+Is a proprietary file format generated and processed by miRDeep2. It contains information of reads mapped to a reference genome. Each line in such a file contains 13 columns:
+
+1. read identifier
+2. length of read sequence
+3. start position in read sequence that is mapped
+4. end position in read sequence that is mapped
+5. read sequence
+6.identifier of the genome-part to which a read is mapped to. This is either a scaffold id or a chromosome name
```
Comment (TorHou) [inline tools/rna_tools/miRDeep2/miRDeep2/mirdeep2.xml]: 6.identifier -> 6. identifier

OCD Torsten is on fire

File: tools/rna_tools/miRDeep2/miRDeep2_mapper/mapper.xml [hunk 4]
```diff
@@ -184,20 +184,33 @@ process and map reads to a reference genome
...
+
+arf format:
+Is a proprietary file format generated and processed by miRDeep2. It contains information of reads mapped to a reference genome. Each line in such a file contains 13 columns:
+
+1. read identifier
+2. length of read sequence
+3. start position in read sequence that is mapped
+4. end position in read sequence that is mapped
+5. read sequence
+6.identifier of the genome-part to which a read is mapped to. This is either a scaffold id or a chromosome name
```
Comment (TorHou) [inline tools/rna_tools/miRDeep2/miRDeep2_mapper/mapper.xml]: another missing blank 
... I'll show myself out

**Comment by TorHou** [comment]
:+1:

File: tools/antismash/antismash.xml [hunk 5]
```diff
@@ -242,16 +240,19 @@ Of course, some regions important to secondary metabolism may have been missed i
...
-
-Marnix H. Medema, Kai Blin, Peter Cimermancic, Victor de Jager, Piotr Zakrzewski, Michael A. Fischbach, Tilmann Weber,
-Rainer Breitling and Eriko Takano (2011). antiSMASH: Rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters. Nucleic Acids Research, doi: 10.1093/nar/gkr466.
 
-http://antismash.secondarymetabolites.org/help.html
+**References**
 
+[Antismash](http://antismash.secondarymetabolites.org/help.html)
 
 Bjoern A. Gruening: https://github.com/bgruening/galaxytools/tree/master/antismash
```
Comment (bgruening) [inline tools/antismash/antismash.xml]: This can also be removed, right?

File: tools/diamond/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+
+name: Diamond
```
Comment (bgruening) [inline tools/diamond/.shed.yml]: "diamond"

File: tools/rna_tools/miRDeep2/miRDeep2_mapper/mapper.xml (hunk 4 above)
Comment (erxleben) [inline tools/rna_tools/miRDeep2/miRDeep2_mapper/mapper.xml]: Thanks for reviewing :-)

**Comment by erxleben** [comment]
all typos removed and comments taken into account

**Comment by bgruening** [comment]
Great, thanks @erxleben and @TorHou!

**Pull request merged**